client/src/app/index.jsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-
-import './styles/index.scss';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect
+} from 'react-router-dom';
 
 import configureStore from './model/store';
-import Main from './components/Main';
+import Dashboard from 'modules/dashboard/components/Dashboard';
+import Main from 'modules/main/components/Main';
+import Cohort from 'modules/cohort/components/Cohort';
+import ShoppingList from 'modules/shopping-list/components/ShoppingList';
 
-const store = configureStore();
+import './styles/index.scss';
 
-ReactDOM.render(
+const store = configureStore();
+const eoc = (
   <Provider store={store}>
-    <Main />
-  </Provider>,
-  document.getElementById('root')
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Main} />
+        <Route path="/dashboard" component={Dashboard} />
+        <Route path="/cohort/:id(\w+)" component={Cohort} />
```

Comment:
what does (\w+) mean in path param?

Reply:
it's regexp which validates id param, accepted values: alphanumeric characters and _, if the validation fails, the main page will be load: <Redirect to="/" />